xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.h

```diff
@@ -243,29 +243,29 @@ class CDVDDemux
   /*
    * Reset the entire demuxer (same result as closing and opening it)
    */
-  virtual void Reset() = 0;
```

Comment:
Think this was some sort of rebase stupidity and lack of attention from my part. Oddly, after rebasing today it went away by itself.